javascript/.codesandbox/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+  "setupTasks": [{ "name": "Install Dependencies", "command": "yarn install" }],
+  "tasks": {
+    "start": {
+      "name": "start",
+      "command": "yarn start",
+      "runAtStart": true,
+      "preview": { "port": 1234 }
+    },
+    "build": { "name": "build", "command": "yarn build", "runAtStart": false }
```

Comment:
For this template, I would also add an install task to make it more discoverable for our users how to install dependencies.